.gitignore

```diff
@@ -1,23 +1,20 @@
 tmp
 examples
 node_modules
-static/js/config-jodi.js
-builds/bfe.dev.js.typeahead
+
 jodi.html
-static/profiles/bibframe/Test profile (jodiw).json
+test.html
 
-builds/bfe.dev.bak
-builds/bfe.jodi.bak
-builds/bfe.jodi.js
-static/profiles/bibframe/Test All (jodiw).json
-static/profiles/bibframe/profiles.json
+builds/*
 
-builds/bfe.test.js
+static/js/config-jodi.js
 static/js/config-test.js
+
+static/profiles/bibframe/Test All (jodiw).json
```

Comment:
Similarly here, could we just ignore everything under `static/profiles/` and make exceptions for whatever test fixture ones we need?

Reply:
All the profiles are gone from static/profiles since they are all in Verso.  I'm going to fix this in the next commit.